jumanji/environments/logic/rubiks_cube/env_viewer.py

```diff
@@ -21,61 +21,20 @@
 import jumanji.environments
 from jumanji.environments.logic.rubiks_cube.constants import Face
 from jumanji.environments.logic.rubiks_cube.types import State
+from jumanji.viewer import Viewer
 
 
-class RubiksCubeViewer:
-    """Abstract viewer class to support rendering and animation"""
-
-    def __init__(self, cube_size: int):
-        self.cube_size = cube_size
-
-    def render(self, state: State) -> None:
-        """Render frames of the environment for a given state using matplotlib.
-
-        Args:
-            state: `State` object corresponding to the new state of the environment.
-        """
-        raise NotImplementedError
-
-    def animate(
-        self,
-        states: Sequence[State],
-        interval: int,
-        save_path: Optional[str],
-    ) -> matplotlib.animation.FuncAnimation:
-        """Create an animation from a sequence of environment states.
-
-        Args:
-            states: sequence of environment states corresponding to consecutive timesteps.
-            interval: delay between frames in milliseconds, default to 200.
-            save_path: the path where the animation file should be saved. If it is None, the plot
-                will not be saved.
-
-        Returns:
-            Animation that can be saved as a GIF, MP4, or rendered with HTML.
-        """
-        raise NotImplementedError
-
-    def close(self) -> None:
-        """Perform any necessary cleanup.
-
-        Environments will automatically :meth:`close()` themselves when
-        garbage collected or when the program exits.
-        """
-        raise NotImplementedError
-
-
-class DefaultRubiksCubeViewer(RubiksCubeViewer):
+class DefaultRubiksCubeViewer(Viewer[State]):
```

Comment:
Should we rename this `RubiksCubeViewer` as it is obviously the default because the only implemented? 
```suggestion
class RubiksCubeViewer(Viewer[State]):
```

Reply:
Good idea

Reply:
This is also consistent with other environments